CONTRIBUTING.md

```diff
@@ -34,6 +34,7 @@ The usual contributing steps are:
   build of all packages and then watch for changes to packages' source code and
   compile these changes on-the-fly as you work.
 * Install [gatsby-dev-cli](/packages/gatsby-dev-cli/) globally: `yarn global add gatsby-dev-cli`
+* Run `yarn install` in each of the sites you're testing with.
```

Comment:
If you run `yarn install` first, then `gatsby-dev` will copy over correctly all your source updates.

Reply:
:+1: